.yarn/versions/31194041.yml

```diff
@@ -0,0 +1,34 @@
+releases:
+  "@yarnpkg/cli": minor
+  "@yarnpkg/core": minor
```

Comment:
The commit title says `fix` which makes this a `patch` but here it's marked as `minor`, it can't be both.

Reply:
I wasn't aware of the convention. I clicked through some other open PRs and the first 3 other `fix(` titled PRs also just update `patch`.
I don't remember seeing suggested title prefixes in the contributing guide. Is there a documented recommended list?